source/agora/consensus/state/UTXOSet.d

```diff
@@ -190,6 +235,130 @@ public class UTXOSet
         log.trace("findUTXO: utxo_hash {} not found", utxo);
         return false;
     }
+
+    /// Ditto
```

Comment:
Those are not `/// Ditto`, since `/// Ditto` means "same as last symbol". Just `///` to make sure they are documented is enough.